### DISCUSSION TITLE
Subscr Subscr Subscr.....forget it, can't see the word appear anymore

### DISCUSSION THREAD

**Discussion by ShonkaiDJ** [OpnForm/OpnForm#451] [General]
Hey guys, I like your software.

It's not just that. I think it's pretty well documented so I see there's someone who not only masters neat code, but also human language pretty well. And somewhere in all the lines of human language I read some words that imply there is a deep understanding of the reason why there is open source software. Sometimes you just need a few words to show 'philosophy' is at play.

I, we, have that deeper understanding too as we are developing some extended 'platform'. Not very public and yes we use a lot of open source software. Think I am going to use your software too but you can forget I am ever going to connect a little tube from my 'credit' account to yours to suck and suck. I like to own. Or better even share. If I get something for my efforts to be, the good people around me do too. I sincerely hate all the little governments that want to have 'just a little' from me because I'm there.

So this message or 'discussion' (I like conversations much more than discussions and debates), this conversation I started to start with. to tell you I will soon add my 50cts to your effort. And furthermore I'd like to see if there are any other creative thoughts of having a revenue-model. Compared to the time we've spend setting up a demonstration model of our extensive plans we are now very close to a moment where we should start think about the credits to keep things running and possible to free up some life-space in time.

Let me know if you have thought of other ways than the popular but unsustainable (damage to decency freedom and life itself) model of having subscr.....pff....sub...tss... well, you know. The 'work for me cause I am smart' model.

I have to end this first message with conveying a deeply felt compliment I started with too. I like your software. It will add at least somewhere between 0,05% and 0,25% of freeing time from the hassles of life for me upcoming year. And extremely smart, devilish, how it's been made clear I can host but not hone a service in my self owned and more and more honed environment.

I think I should start thinking of these kind of ways too, but I can't get myself to do that. I rather cooperate and do a straight investment of exchanging quality energy leaving a timeless impression.

**Comment by ShonkaiDJ** [comment]
I think I am right about the good things and very wrong about the bad conclusions and assumptions I made here. Discussion closed.

**Comment by JhumanJ** [comment]
Thank you for your kind words and thoughtful message! I appreciate your perspective on subscription models. It’s important to note that subscriptions are only for our cloud version, which involves costs beyond just time for us.

Despite not being profitable with this project, we continue passionately because we believe there is a real need for a great open-source form builder. We’re always open to exploring alternative monetization strategies, so if you have ideas, I’d love to discuss them further.

Thanks again for your support and for considering our software!

Best regards,
Julien
